### ISSUE TITLE
Mob class Completed

### ISSUE THREAD

**Issue by 133794m3r** [133794m3r/itp298_capstone#1]
This issue will track the creation of a mob class and the methods required of it.
- [ ] Constructor
- [ ] Deconstructor
Private properties
- [ ] Attack, Defense, Life, Name, XP
- [ ] Loot table linked blocked by
Public methods
- [ ] Damaged
- [ ] Announce

Nice to have
- [ ] Mob has the ability to call different attacks of different strengths.
- [ ] Mob can use an item.
